src/main/java/com/mkul/game/ox/BoardDraw.java

```diff
@@ -0,0 +1,32 @@
+package com.mkul.game.ox;
+
+class BoardDraw {
+
+    private final int boardSize;
+
+    BoardDraw(int boardSize) throws IncorrectBoardSizeException {
+        if (boardSize < 1)
+            throw new IncorrectBoardSizeException();
+        this.boardSize = boardSize;
+    }
+
+    void show(BoardScore boardScore) {
+        System.out.printf("%-3s", " ");
+        for (int x = 0; x < boardSize; x++) {
+            System.out.printf("%-3s ", x);
+        }
+        System.out.print("\n");
+
+        for (int x = 0; x < boardSize; x++) {
+            System.out.printf("%-2s ", x);
+            for (int y = 0; y < boardSize; y++) {
+                if (boardScore.isFieldExist(x, y)) {
+                    System.out.printf("%-4s", boardScore.getValue(x, y).toString());
+                } else {
+                    System.out.printf("%-4s", "_");
+                }
+            }
+            System.out.print("\n");
+        }
```

Comment:
I think that code could be nicer without for loops within for loops.

Reply:
You wish to put inner loop into method? or have you other preferences :D ?